site/hxapi/asynchronous-bookings.md

```diff
@@ -0,0 +1,143 @@
+---
+
+---
+
+# Asynchronous Bookings
+
+## Background
+
+We now offer products that use an asynchronous booking process. This allows partners to handle bookings for a wider range of products. However, asynchronous bookings require a different approach compared to synchronous bookings to provide a seamless experience for customers.
```

Comment:
"We now offer":
Are we planning to merge this change soon?
Or hold it back until we have actually implemented it?

Reply:
Also wondering whether it would benefit from an extra introductory sentence to explain why we need this, as more motivation for why the partner should bother implementing it.

Reply:
> "We now offer": 

Removed the link to this page from the parent page. 

> Also wondering whether it would benefit from an extra introductory sentence

Added this in